dts/bindings/sensor/st,iis2dlpc-common.yaml

```diff
@@ -11,6 +11,17 @@ properties:
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
 
+    drdy-pin:
```

Comment:
How about `drdy-int`?  `foo-pin` is currently used in Nordic boards to identify a GPIO ordinal; I could also see it being used in other contexts as a pad (pin) identifier.

Alternatively, since there are only two choices:  `drdy-int2` as a boolean property; if not selected then it's `INT1`.  This would eliminate the need to add build-time value validation.  (Though so would making this an int enum accepting only 1 and 2.

Reply:
> How about `drdy-int`? `foo-pin` is currently used in Nordic boards to identify a GPIO ordinal; I could also see it being used in other contexts as a pad (pin) identifier.
> 
drdy-int seems a better name, thanks!

> Alternatively, since there are only two choices: `drdy-int2` as a boolean property; if not selected then it's `INT1`. This would eliminate the need to add build-time value validation. (Though so would making this an int enum accepting only 1 and 2.

To be honest I don't like the idea of having a boolean like that. It seems to me more clear an attribute that specify either 1 or 2. And yes, I will add the two possible values using the 'enum' declaration.

Does it make sense to you?

Reply:
That'd be fine; the main thing is to make sure people can't configure it to an invalid state.

Reply:
When specifying enum a build-time checking, right? At least I tried with 'bad' values and didn't build up.